Database/DatabaseConfig.py

```diff
@@ -4,18 +4,39 @@
 
 load_dotenv(find_dotenv())
 
-connectionString = os.environ.get("MONGODB_CONNECTION_STRING")
-
-dbclient = MongoClient(connectionString)
-
-beehive = dbclient.beehive
-
-
-def get_beehive_user_collection():
-    return beehive.users
-
-def get_beehive_image_collection():
-    return beehive.images
-
-def get_beehive_admin_collection():
-    return beehive.admins
\ No newline at end of file
+class DatabaseConfig:
+    _instance = None
+    _db = None
+    _client = None
+    
+    @classmethod
+    def get_database(cls):
+        if cls._db is None:
+            connection_string = os.environ.get("MONGODB_CONNECTION_STRING", "mongodb://localhost:27017/")
```

Comment:
I don't think we need to explicitly state mongodb://localhost:27017/ here. Can you check?